apps/web/src/features/myAccounts/index.tsx

```diff
@@ -25,12 +27,28 @@ const MyAccounts = ({ safes, onLinkClick, isSidebar = false }: MyAccountsProps)
   const [searchQuery, setSearchQuery] = useState('')
   useTrackSafesCount(safes, wallet)
 
+  const safeDeploymentProps = useMemo(() => {
+    return {
+      account: {
+        owners: wallet?.address ? [wallet.address] : [],
+        threshold: 1,
+      },
+      saltNonce: Date.now().toString(),
+      dryRun: true,
+    }
+  }, [wallet?.address])
+
+  const hasSafenetFeature = useHasSafenetFeature()
+  const { status: safenetDeployDryRunStatus } = useDeploySafenetAccountQuery(safeDeploymentProps, {
+    skip: !hasSafenetFeature,
+  })
```

Comment:
Lets move this to a new `useIsWalletSafenetAllowListed` hook or something similar.